contracts/compliance/LimitHolder.sol

```diff
@@ -111,8 +115,11 @@ contract LimitHolder is ICompliance, AgentRole {
 
     function created(address _to, uint256 _value) public override onlyAgent returns (bool) {
         require(_value > 0, "No token created");
-        updateShareholders(_to);
-        return true;
+        if (holderCount() < holderLimit) {
```

Comment:
that's not the role of the `created()` function to check if the transaction can be processed, you have to use the `canTransfer()` for that. `created()` `destroyed()` and `transferred()` are called only to update the shareholders list in this case, or to do some other stuffs in different compliance contracts post-transaction. in the case of a minting process you should just use the `canTransfer()` function with `_msgSender()` as a `_from`